src/contractors.jl

```diff
@@ -72,20 +69,69 @@ function 𝒩{T}(f, f′, X::Interval{T})
 end
 
 
-IntervalArithmetic.mid(X::IntervalBox, α) = mid.(X, α)
-
 doc"""
 Multi-variable Newton operator.
 """
 function 𝒩(f::Function, jacobian::Function, X::IntervalBox)  # multidimensional Newton operator
-
     m = IntervalBox(Interval.(mid(X, where_bisect)))
     J = jacobian(SVector(X))
 
     return IntervalBox(m - (J \ f(m)))
 end
 
 
+# Krawczyk
+struct Krawczyk{F, FP} <: Contractor{F}
+    f::F
+    f′::FP   # use \prime<TAB> for ′
+end
+
+function (C::Krawczyk)(X, tol)
```

Comment:
This function is (almost?) exactly the same as the equivalent for Newton, which is why my idea had been to combine the two into a single type. It's not necessary to use a single type, but I think this function and the equivalent for Newton should be able to be combined.

Reply:
E.g. define a function `operator(C::Krawczyk) = 𝒦`.
I *think* this should be efficient.